packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -412,7 +463,7 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
         });
         registry.registerMenuAction(NavigatorContextMenu.CLIPBOARD, {
             commandId: FileNavigatorCommands.COPY_RELATIVE_FILE_PATH.id,
-            label: 'Copy Relative Path',
+            label: FileNavigatorCommands.COPY_RELATIVE_FILE_PATH.label,
```

Comment:
I believe it can be omitted if the `label` is the same as the `command` definition.
The same is true for other occurences.

Reply:
Also good to know, I've removed the labels from the menu items